### PR TITLE
HW8 Task02

### PR DESCRIPTION
**Pull request by LinaKrevetochka**

### CODE REVIEW

File: hw/hw_08_task_02.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import sqlite3
+from typing import Tuple
+
+
+# flake8: noqa: S608
```
Comment (Cjkjvfnby) [inline hw/hw_08_task_02.py:14]: We should not disable it. This indicates a real issue.  Do you know about Little Bobby Tables?

File: hw/hw_08_task_02.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+class TableData:
+    def __init__(self, database_name: str, table_name: str):
+        self.database_name = database_name
+        self.table_name = table_name
+
+    @staticmethod
+    def connect_database(database_name: str) -> sqlite3.Cursor:
+        conn = sqlite3.connect(database_name)
```
Comment (Cjkjvfnby) [inline hw/hw_08_task_02.py]: Which method is always called in examples and never in this code?

File: hw/hw_08_task_02.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    def __getitem__(self, item: str) -> Tuple:
+        cursor = TableData.connect_database(self.database_name)
+        cursor.execute("SELECT * from " + self.table_name + " WHERE name =?", (item,))
+        return cursor.fetchone()
+
+    def __contains__(self, item: str) -> bool:
+        cursor = TableData.connect_database(self.database_name)
+        cursor.execute("SELECT * from " + self.table_name + " WHERE name =?", (item,))
+        return False if cursor.fetchone() is None else True
```
Comment (Cjkjvfnby) [inline hw/hw_08_task_02.py]: Can we simplify it?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: hw/hw_08_task_02.py (hunk 1 above)
Comment (LinaKrevetochka) [reply hw/hw_08_task_02.py:14]: I had no idea about it. I have googled it just now, the comics is funny :) I'll fix it.

File: hw/hw_08_task_02.py (hunk 2 above)
Comment (LinaKrevetochka) [reply hw/hw_08_task_02.py]: I'm not sure, but maybe you are talking about closing connection?

Comment (Cjkjvfnby) [reply hw/hw_08_task_02.py]: Exactly!

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**